domain-models-runtime/src/test/java/org/folio/DemoRamlRestTest.java

```diff
@@ -344,6 +345,38 @@ public void invalidCqlClosesConnection(TestContext context) {
     }
   }
 
+  /**
+   * Check that all Set-Cookie entries are returned.
+   *
+   * <p>See {@link org.folio.rest.impl.BooksDemoAPI#postRmbtestsBooks}
+   * @throws JsonProcessingException
+   */
+  @Test
+  public void testMultiHeaders() throws JsonProcessingException {
+    Data d = new Data();
+    d.setAuthor("a");
+    d.setGenre("g");
+    d.setDescription("description");
+    d.setTitle("title");
+    d.setDatetime(new Datetime());
+    d.setLink("link");
+
+    Book b = new Book();
+    b.setData(d);
+    b.setStatus(0);
+    b.setSuccess(true);
+
+    ObjectMapper om = new ObjectMapper();
+    String book = om.writerWithDefaultPrettyPrinter().writeValueAsString(b);
```

Comment:
The duplicate code here, in  getBookWithRoutingContext(TestContext) and test(TestContext) should be extracted into a helper method, for example `private static String book()`.

Reply:
ok. done